plugins/uploader/download.sh

```diff
@@ -63,13 +63,14 @@ fi
 
 url=$host/v0/organizations/$organization/clusters/$cluster/data/$datatype/config
 set +x
+status=0
 results=$(curl -X POST $url \
   -L \
   -H "Authorization: Bearer ${FAIRWINDS_TOKEN//[$'\t\r\n']}" \
   -H "Accept: application/x-yaml" \
   -O $file \
-  --fail 2>&1)
-if [ $? -ne 0 ]
+  --fail 2>&1) || status=$?
+if [ $status -ne 0 ]
 then
     echo $results
     # If the response isn't a 404 this job will fail
```

Comment:
should the grep below be `grep -v` (inverted)? Not sure how --quiet works

Reply:
It feels right to me, but maybe it's just too early. That should fail if it finds something other than 404. That way a 200 would just skip that if statement and output a file. a 404 would exit successfully and anything other than a 404 would throw an error and fail the pod.

Reply:
ohh I understand now. 👍